static/dtale/DataViewer.jsx

```diff
@@ -23,6 +23,7 @@ import { ColumnMenu } from "./column/ColumnMenu";
 import { exports as gu } from "./gridUtils";
 import Descriptions from "./menu-descriptions.json";
 import { DataViewerMenu } from "./menu/DataViewerMenu";
+import { MenuTooltip } from "./menu/MenuTooltip";
```

Comment:
In order to fix horizontal scroll issue when the menu is pinned please add the following to this file:
* Line 243: `width={width - (this.props.menuPinned ? 198 : 3)}`
* Line 284: `menuPinned: PropTypes.bool,`
* Line 288: `({ dataId, iframe, theme, settings, menuPinned }) => ({ dataId, iframe, theme, settings, menuPinned }),`